meson_options.txt

```diff
@@ -426,6 +426,8 @@ option('efi-libdir', type : 'string',
        description : 'path to the EFI lib directory')
 option('efi-includedir', type : 'string', value : '/usr/include/efi',
        description : 'path to the EFI header directory')
+option('efi-tpm-pcr-compat', type : 'boolean', value : 'false',
```

Comment:
Shouldn't this default to true as we introduce it, and then changed to false later, to reduce the risk of making systems unbootable?

Reply:
or in other words: do as i say, don't do what i do ;-)